test/performance/README.md

```diff
@@ -1,128 +1,189 @@
+   Then set ``--kubeconfig`` flag to the path of the kubeconfig file of your cluster and use the newly added Kubestellar provider (`--provider=ks`) and run the following command to create your workload:
 
+   ```bash
+   $ cd $WORK_DIR
+   $ go run cmd/clusterloader.go --testconfig=./testing/load/performance-test-config.yaml --kubeconfig=<path>/wds-kubeconfig --provider=ks --v=2
+   ```
 
-3. Run your first workload:
 
 
-Set ``--kubeconfig`` flag to the path of the kubeconfig file of your cluster (e.g., WDS kubeconfig) and use the newly added Kubestellar provider
+5. Clean up the generated workload Kubernetes API objects from your cluster:
 
-```bash
-$ cd perf-tests/clusterloader2/
-$ go run cmd/clusterloader.go --testconfig=./testing/load/performance-test-config.yaml --kubeconfig=<path>/wds-kubeconfig --provider=ks --v=2
-```
+   ```bash
+   $ cd $WORK_DIR
+   $ ./cleanup.sh
+   ```
 
 
-## Collect the measurements 
+### b) Collect the creation & update timestamps for workload benchmark (TO BE REVISED: WORK-IN-PROGRESS)
 
-To collect the creation and status update for a k8s object proceed as following:  
+To collect the creation and status update for the workload benchmark objects proceed as following:  
 
 1. Create a kubeconfig with contexts for the `wds1`, `its1` and `wec1`:
 
-For example: 
+   For example: 
 
-```bash
-$ kubectl config get-contexts
-CURRENT   NAME     CLUSTER                                          AUTHINFO                                                                 NAMESPACE
-          its1     its1-cluster                                     its1-admin                                                                default
-          wec1     <url>:port#                                      <defaul-value>                                                            default
-*         wds1     wds1-cluster                                     wds1-admin                                                                default
-```
+   ```bash
+   $ kubectl config get-contexts
+   CURRENT   NAME     CLUSTER                                          AUTHINFO                                                                 NAMESPACE
+            its1     its1-cluster                                     its1-admin                                                                default
+            wec1     <url>:port#                                      <defaul-value>                                                            default
+   *         wds1     wds1-cluster                                     wds1-admin                                                                default
+   ```
 
-Use the following command to rename the default context name for `wec1` workload execution cluster:
+   Use the following command to rename the default context name for `wec1` workload execution cluster:
```

Comment:
It does not make sense to tell the reader to rename a context after she already has contexts with the needed names. If I understand correctly, your intent here is to give some help about how the user might get into the state described just above. If so then that should be indicated. Perhaps by starting this remark with something like "To get into that state you may want to rename some existing contexts. You can do that ...".

Reply:
Please, do revise yet the content of the subsection: **Collect the creation & update timestamps for workload benchmark** - It's a working in progress and I will revised all the contents. I am going to DELETE the content of the subsection for now to avoid any confusion as it will be update in a NEW PR.